OpenGLTechTalks/Basic/src/Shader.h

```diff
@@ -0,0 +1,38 @@
+#ifndef SHADER
+#define SHADER
+
+#include <glad/glad.h>
+
+#include <string>
+
+namespace core
+{
+    class Shader
+    {
+    private:
+        unsigned int m_ID;
+    public:
+        Shader();
+
+        Shader(const char* vertexPath, const char* fragmentPath);
+
+        ~Shader();
+
+        void activate();
+
+        void deActivate();
```

Comment:
deactivate or instread of deActivate.
Or create method [ void SetActive(bool active) ]